src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir-Book", sans-serif;
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy", sans-serif;
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+  
+.search-form {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin: auto;
+  margin-top: 20px;
+}
+  
+.input {
+  position: relative;
+  padding: 20px 60px;
+  border: none;
+  border-radius: 4px;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  font-family: "Avenir-Book", sans-serif;
+  line-height: 22px;
+  letter-spacing: 0.6px;
+  font-size: 16px;
+}
+
+.search-form-small {
+  height: 42px;
+}
+  
+.input-small {
+  padding-left: 33px;
+  font-family: "Avenir-Book", sans-serif;
+  line-height: 20px;
+  font-size: 14px;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input-small::placeholder {
+  font-size: 14px;
+}
+
+.search-form::after {
+  content: "";
+  background-image: url(images/Search.svg);
+  background-size: 19px;
+  display: inline-block;
+  position: absolute;
+  left: 26px;
+  top: calc(50% - 11px);
+  text-align: center;
+  line-height: 22px;
+  width: 19px;
+  height: 19px;
+}
+  
+.search-form-small::after {
+  position: absolute;
```

Comment:
You have this property in .search-form::after so you don't need to repeat it here.

Reply:
I have fixed the issue with on line 77. Thank you. 
As for the failed tests, I can see that most likely the issue is with font, however, all the attempts i've made gave no result. 
Is there anything you could recommend?

Reply:
fixed what i could for now. now at least it passes all the tests.